shared/src/webMain/resources/index.html

```diff
@@ -149,15 +149,36 @@
         <script type="application/javascript">
             const body = document.body
 
-            const hasWasmGcSupport = () =>
-                WebAssembly.validate(new Uint8Array([0, 97, 115, 109, 1, 0, 0, 0, 1, 5, 1, 95, 1, 120, 0]))
-
-            const hasWasmExceptionSupport = () =>
-                WebAssembly.validate(new Uint8Array([0, 97, 115, 109, 1, 0, 0, 0, 1, 4, 1, 96, 0, 0, 3, 2, 1, 0, 10, 8, 1, 6, 0, 6, 64, 25, 11, 11])
-                );
-
-            const hasWasmReferenceTypeSupport = () =>
-                WebAssembly.validate(new Uint8Array([0, 97, 115, 109, 1, 0, 0, 0, 1, 4, 1, 96, 0, 0, 3, 2, 1, 0, 10, 7, 1, 5, 0, 208, 112, 26, 11]))
+            /*
+            Represents the next Wasm module:
+            (module
```

Comment:
A command line how to compile it also would be helpful.

Reply:
Done 👌